micro-benchmarks/DRB169-nowait-orig-no.c

```diff
@@ -0,0 +1,53 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+
+/**
+ * CG.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 CG suit.
+ * Tsan and Intel Inspector can not correctly analysis the nowaite, and report a false postive.
```

Comment:
typo "nowaite" --> "nowait"

Reply:
Suggestion, instead of mentioning which tool will report and which one won't, better add in the description the constructs and clauses and how are they solving issues (if any). Anyways, in the report, we will be mentioning about the tools' capability.